applications/nrf_desktop/doc/dfu_mcumgr.rst

```diff
@@ -77,6 +62,61 @@ Make sure that the DFU lock utility is enabled if your nRF Desktop application c
 In other words, you cannot simultaneously enable the :ref:`CONFIG_DESKTOP_DFU_MCUMGR_ENABLE <config_desktop_app_options>` option and the :kconfig:option:`CONFIG_CAF_BLE_SMP` Kconfig option.
 
+You have to choose backend for the DFU process by enabling one of the following options:
+
+  * :kconfig:option:`CONFIG_DESKTOP_DFU_BACKEND_MCUBOOT` - This option depends on the MCUboot bootloader support for the DFU process.
+    For details, see :ref:`mcuboot` in the Zephyr documentation.
+  * :kconfig:option:`CONFIG_DESKTOP_DFU_BACKEND_SUIT` - This option depends on the SUIT support for the DFU process.
+    For details, see :ref:`ug_nrf54h20_suit_dfu` in the Zephyr documentation.
+
+.. note::
+    B0 bootloader is not integrated with MCUmgr.
+
+.. _nrf_desktop_dfu_mcumgr_mcuboot:
+
+MCUmgr configuration using MCUboot backend
+==========================================
+
+The MCUboot backend configuration selects the following options:
+
+* MCUmgr groups:
+
+  * :kconfig:option:`CONFIG_MCUMGR_GRP_IMG` - This option enables the MCUmgr image management handlers, which are required for the DFU process.
+    For details, see :ref:`zephyr:device_mgmt` in the Zephyr documentation.
+  * :kconfig:option:`CONFIG_MCUMGR_GRP_IMG_MUTEX`
+  * :kconfig:option:`CONFIG_MCUMGR_GRP_OS` - This option enables the MCUmgr OS management handlers, which are required for the DFU process.
+    For details, see :ref:`zephyr:device_mgmt` in the Zephyr documentation.
+
+* :kconfig:option:`CONFIG_IMG_MANAGER`
+* :kconfig:option:`CONFIG_STREAM_FLASH`
+* :kconfig:option:`CONFIG_MCUBOOT_BOOTUTIL_LIB`
+
+To use the module with this backend, you must also enable the :kconfig:option:`SB_CONFIG_BOOTLOADER_MCUBOOT` Kconfig option.
+
+By default, the MCUmgr DFU module confirms the image using the :c:func:`boot_write_img_confirmed` function during the system boot.
+If the :kconfig:option:`CONFIG_DESKTOP_DFU_MCUMGR_MCUBOOT_DIRECT_XIP` option is enabled, the MCUmgr DFU module assumes that the bootloader simply boots the image with a higher version and does not confirm the newly updated image after a successful boot.
+Make sure that :kconfig:option:`CONFIG_DESKTOP_DFU_MCUMGR_MCUBOOT_DIRECT_XIP` Kconfig option is enabled if devices use the MCUboot bootloader in direct-xip mode without revert.
+The option is enabled by default if :kconfig:option:`CONFIG_MCUBOOT_BOOTLOADER_MODE_DIRECT_XIP` is enabled.
+
+.. _nrf_desktop_dfu_mcumgr_suit:
+
+MCUmgr configuration using SUIT backend
+=======================================
+
+The SUIT backend configuration requires the following options:
+
+* :kconfig:option:`CONFIG_MGMT_SUITFU` - This option enables the SUIT Firmware Update Management that implements MCUmgr image management handlers required for the DFU process.
+* :kconfig:option:`CONFIG_MGMT_SUITFU_GRP_SUIT` - This option is needed to upload information about all manifests present on the device.
+* :kconfig:option:`CONFIG_SSF_SUIT_SERVICE_ENABLED`
```

Comment:
Shouldn't we move it under `SUIT Firmware Update Management dependencies`? Or maybe we can simply merge both sections together?